### ISSUE TITLE
0.8.0 not working with intern-client

### ISSUE THREAD

**Issue by iyegoroff** [SitePen/remap-istanbul#105]
Running following command 
```
$ intern-client config=intern reporters=node_modules/remap-istanbul/lib/intern-reporters/JsonCoverage
``` 
results in error:

```
Error: Attempt to require unloaded module istanbul
  at contextRequire  <node_modules/dojo/loader.ts:345:2>
  at req  <node_modules/dojo/loader.ts:124:2>
  at <node_modules/remap-istanbul/src/intern-reporters/JsonCoverage.js:1:22>
  at ContextifyScript.Script.runInThisContext  <vm.js:25:33>
  at Object.exports.runInThisContext  <vm.js:77:17>
  at <node_modules/dojo/loader.ts:838:14>
  at tryToString  <fs.js:425:3>
  at FSReqWrap.readFileAfterClose [as oncomplete]  <fs.js:412:12>
```
intern.js here is just an empty config file, intern version - 3.4.2.
After downgrading to version 0.7.0 test command completed without this error.

**Comment by jason0x43** [comment]
That kind of error generally means that an AMD loader is trying to load a node module. Probably omething is doing `var istanbul = require(’istanbul')` and the AMD loader isn't handling it in CJS compatibility mode.

**Comment by jdonaghue** [comment]
Thanks @jason0x43! I think you are right, that seems to be exactly what is happening. I will see if I can get this fixed this morning.

**Comment by jdonaghue** [comment]
@iyegoroff we have just released and published `remap-istanbul@0.8.1` can you please pull down and verify that this fixes your issue? Thanks!

**Comment by iyegoroff** [comment]
Thanks, @jdonaghue! That error is gone now.
